### ISSUE TITLE
Changes made to css and scss files aren't being reflected

### ISSUE THREAD

**Issue by samar14641** [howdyai/botkit#1626]
I modified style.css to change some properties (styling for a new class, colours, border style, backgrounds etc.), but on restarting the server on localhost, none of the changes were reflected. 

Changes made to _chat.scss aren't being reflected either. 

- Botkit version: 0.7.0
- Messaging Platform: Web
- Node version: 8.11.1
- OS: Windows 8.1

**Comment by benbrown** [comment]
If you change the .scss files, you have to first rebuild them into CSS for the changes to take effect. Instructions are in the readme.

If you edit styles.css directly, it should work. That file is just being served up statically. Clear your cache and make sure you see the changes in the browser if you load the file directly.

**Comment by samar14641** [comment]
I had recompiled the .scss files as mentioned in the readme and cleared the cache.

In the _socket.addEventListener_ of _client.js_, I am checking the message.text for a particular pattern, and if the condition is satisfied, I switch the type from _message_ to _m2m_. 
```
if (regexPattern.test(message.text))
{
    console.log('switch type');
    message.type = 'm2m';  // This switch is happening as expected
}
that.trigger(message.type, message);
````
As the style is defined for class _message_, I added a similar styling for class _m2m_ in __chat.scss,_ but the div doesn't show, and the conversation carries on. 

This screenshot shows that message.type has been changed successfully: 

![botkitMessage](https://user-images.githubusercontent.com/33169942/56782730-fff6cb80-6805-11e9-80d4-e8198df0c2f6.JPG)

Styling: 
````
.m2m {
    font-size: 14px;
    padding: 0.5rem;
    background: #e5dcdb;
    color: #000000;
    border-radius: 5px;
    width: auto;
    display: block;
    max-width: 75%; }
````

The reason I'm switching message.type in some cases is because in _chat.html_, I'm adding a canvas to the bottom of the div using custom Handlebars. 

````
{{#ifCond message.type '==' "m2m"}}
    <canvas id="messageCanvas"></canvas>
{{/ifCond}
````
